cmd/describe/ingress/cmd_test.go

```diff
@@ -143,44 +144,41 @@ Wildcard Policy:            WildcardsAllowed
 
 		It("Cluster not ready", func() {
 			t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, classicClusterNotReady))
-			args := NewDescribeIngressUserOptions()
-			args.ingress = "apps"
-			runner := DescribeIngressRunner(args)
+			runner := DescribeIngressRunner(NewDescribeIngressUserOptions())
 			err := t.StdOutReader.Record()
 			Expect(err).ToNot(HaveOccurred())
-			err = runner(context.Background(), t.RosaRuntime, NewDescribeIngressCommand(), []string{
-				"-c", mockReadyCluster.ID(),
-			})
+			cmd := NewDescribeIngressCommand()
+			cmd.Flag("ingress").Value.Set("apps")
+			cmd.Flag("cluster").Value.Set(mockReadyCluster.ID())
```

Comment:
Just a bit confusing here:
The test is checking the cluster state but the mock cluster builder here has a ready cluster? 
Would another var be used here?

Reply:
https://github.com/openshift/rosa/pull/1984/files/fd6acd3a3494de9b6e921ac87fc19aaacdf8acc7#diff-0ebe25c7c6cfa28f965bf3b8cb5b2f7357684a0f5c9a7fbf71c313b2f6abdb0bR146

The response is injected here as not ready :)

Reply:
personally would have preferred mockCluster instead of mockReadyCluster but works for me :)